docs/src/pages/components/data-grid/rendering/rendering.md

```diff
@@ -273,6 +273,13 @@ This demo replaces it with the [Pagination](/components/pagination/) component.
 
 {{"demo": "pages/components/data-grid/rendering/CustomPaginationGrid.js", "bg": "inline"}}
 
+### Toolbar
+
+You can replace the default toolbar by providing a `header` component.
+This demo showcases how this can be achieve.
+
+{{"demo": "pages/components/data-grid/rendering/CustomToolbarGrid.js", "bg": "inline"}}
+
```

Comment:
I would suggest we document the components in the vertical order they are rendered in the grid, so moving this section before `### Loading overlay`.

Reply:
Done